### PR TITLE
monikerize interfaces and member implements in ECMAXML

### PR DESCRIPTION
**Pull request by TianqiZhang**

### CODE REVIEW

**Comment by TianqiZhang** [comment]
@joelmartinez it seems integration tests failed due to this change, is it because I changed the schema?

**Comment by TianqiZhang** [comment]
> @joelmartinez it seems integration tests failed due to this change, is it because I changed the schema?

Never mind, it turns out that I got the schema wrong. Fixed it.

File: mdoc/Mono.Documentation/MDocUpdater.cs [hunk 1]
```diff
@@ -2839,7 +2846,13 @@ private static void AddImplementedMembers(FrameworkTypeEntry typeEntry, MemberRe
             foreach (var implementedMember in implementedMembers)
             {
                 var value = msxdocxSlashdocFormatter.GetDeclaration(implementedMember);
-                WriteElementWithText(e, "InterfaceMember", value);
+                var interfaceMemberElement = WriteElementWithText(e, "InterfaceMember", value);
+
+                interfaceMemberElement.AddFrameworkToElement(typeEntry.Framework);
+                if (typeEntry.IsMemberOnLastFramework(mi))
```
Comment (joelmartinez) [inline mdoc/Mono.Documentation/MDocUpdater.cs:2852]: Just like the check you added in `UpdateType` for `typeEntry.IsOnFirstFramework`, there should be a similar check here at the member level ... you can use `typeEntry.IsMemberOnFirstFramework(member)` to determine when you should clear the list of interface members (unless this is already happening somewhere and it's just not in these diffs?))

Comment (TianqiZhang) [reply mdoc/Mono.Documentation/MDocUpdater.cs:2852]: oh yes, the logic is already there at line 2808:
```
            var fingerprint = DocUtils.GetFingerprint(mi);
            if (typeEntry.IsMemberOnFirstFramework(mi))
            {
                ClearElement(root, "Implements");
            }
```

Comment (joelmartinez) [reply mdoc/Mono.Documentation/MDocUpdater.cs:2852]: excellent!

**Comment by joelmartinez** [comment]
@TianqiZhang ... how do you want to coordinate this release going out ? Should we merge the return type changes and this to release as 5.7.6?
https://github.com/mono/api-doc-tools/pull/496

**Pull request merged**